internal/pkg/state/load_local_test.go

```diff
@@ -262,10 +262,10 @@ func loadLocalTestState(t *testing.T, m *manifest.Manifest, fs filesystem.Fs) (*
 
 	// Load state
 	assert.NoError(t, err)
-	state, err := New(project.NewWithManifest(context.Background(), fs, m), d)
+	state, err := New(context.Background(), project.NewWithManifest(context.Background(), fs, m), d)
```

Comment:
Na niektorych miestach sa nepredaval `ctx`, tj. vedie to aj k lepsiemu kodu :arrow_right: ked operacie na seba spravne nadvazuju, tak sa aj spravne ukoncia v pripade timeout alebo ineho zrusenia operacie.